tests/unit/logging_test.py

```diff
@@ -2,8 +2,7 @@
 import core
 from core.telemetry_handler import telemetry_handler
 import time
-from core.rovecomm import RoveCommPacket
-from core.rovecomm import ROVECOMM_SUBSCRIBE_REQUEST
+from core.rovecomm_module.rovecomm import RoveCommPacket, ROVECOMM_SUBSCRIBE_REQUEST
```

Comment:
core/telemetry_handler.py and test/unit/rovecomm_test.py both import as `from core import RoveCommPacket`, whereas here you explicitly mention core.rovecomm_module.rovecomm. Am I correct in assuming that this is the correct import and that those test files are outdated?

Reply:
I import RoveCommPacket into core within it's __init__.py, but yes structurally it makes more sense to explicitly reference the submodule